Patches/OpenCV3/Patch.cmake

```diff
@@ -14,3 +14,11 @@ file(COPY ${OpenCV_patch}/graphcuts.cpp
 file(COPY ${OpenCV_patch}/OpenCVFindLibsVideo.cmake
   DESTINATION ${OpenCV_source}/cmake
 )
+
```

Comment:
I split the this branch into two commits so you can see what the patch is actually changing.  The first commit just copies the files unchanged from the source.  The second makes the patch.  The only change here is a couple of lines in two files to add MSVC 2017 to the list of known Windows compilers, so it should not impact other platforms.  The patch is the same as used upstream.